src/validation/rules/custom/NoSchemaIntrospectionCustomRule.ts

```diff
@@ -28,7 +28,9 @@ export function NoSchemaIntrospectionCustomRule(
         context.reportError(
           new GraphQLError(
             `GraphQL introspection has been disabled, but the requested query contained the field "${node.name.value}".`,
-            node,
+            {
+              nodes: node,
+            },
```

Comment:
Please format as single lines.
Same as you did here: https://github.com/graphql/graphql-js/pull/3465/files#diff-1b1ca2cbd9180fc2e2431a40a79598a9e92880d044ee3b2146bf05caf52b6ad6R33
Please check all changes.

Reply:
done.